app/assets/stylesheets/_sign_in.scss

```diff
@@ -0,0 +1,64 @@
+.border_right {
+  padding-right: 10px;
+  border-right: 1px solid #ccc;
+}
+
+.custom_size {
+  font-size: 70px;
+}
+
+.headers {
+  padding-bottom: 30px;
+}
+
+.slogan {
+  padding-top: 70px;
+  padding-bottom: 15px;
+}
+
+hr {
+  width: 60%;
+  text-align: center;
+}
+
+.column_size {
+  height: 400px;
+  width: 50%;
+}
+
+.button_sign_up {
+  text-align: center;
+  padding-top: 23px;
+}
```

Comment:
I'd follow Bootstrap's naming convention, as long as we're using it. This should be `.btn-sign-up`.